### PR TITLE
Update Changelog with "recent" changes

### PR DESCRIPTION
**Pull request by nthykier**
@sunfall, @Xenega: Please let me know if you think it needs additional updates.

I have deliberately been a bit vague on the new items to avoid spoiling.  Let me know if you would rather that we expanded that to include the concrete changes instead. :)

### CODE REVIEW

File: Changelog [hunk 1]
```diff
@@ -1,6 +1,87 @@
...
+  retroactively apply the game data from the latest version
+  of singularity - including making new game content
+  researchable. (Closes: #56, #122, #145) (nthykier)
+- Unify multiple messages into a single dialog with multiple
+  pages.  This new dialog also supports resuming the game
+  instead of forcing the game to be paused. (Xenega)
+- Events can now expire and two non-unique events now do.
+  Previously, they could reappear continously possibly
+  making the game almost impossible (or completely
+  neutering the effected group).  (nthykier)
```
Comment (sunfall) [inline Changelog]: s/effected/affected/

File: Changelog [hunk 2]
```diff
@@ -1,6 +1,87 @@
...
+- The cash display now shows expected cash change over the
```
Comment (sunfall) [inline Changelog]: shows expected/shows the expected/

File: Changelog [hunk 3]
```diff
@@ -1,6 +1,87 @@
...
+  next 24 hours next to the current cash.  Likewise, the
+  cpu display includes how much spare CPU there is after
```
Comment (sunfall) [inline Changelog]: cpu/CPU/

File: Changelog [hunk 4]
```diff
@@ -1,6 +1,87 @@
...
+  maintenance and construction. These numbers also include
+  purchases currently under consideration.
```
Comment (sunfall) [inline Changelog]: Under consideration?  As in, when you've clicked one but haven't actually hit build yet?  Just making sure this is the right word.

File: Changelog [hunk 5]
```diff
@@ -1,6 +1,87 @@
...
+  (Closes: #21, #100) (nthykier)
+- Support searching for a savegame by name in the Load
+  dialog (nthykier)
+- Make the base type and its total CPU visible in the list
+  of bases at a given location.  (Closes: #64) (nthykier)
+- Fix a crash when savegame names contained non-ASCII UTF-8
+  characters (Closes: https://bugs.debian.org/718447)
+  (nthykier)
+- Fix visual lag at highest game speed due to recalculating
```
Comment (sunfall) [inline Changelog]: highest/the highest/

File: Changelog [hunk 6]
```diff
@@ -1,6 +1,87 @@
...
+  the "day/night" layers (nthykier)
+- It is now possible to use double-click as a short-cut of
```
Comment (sunfall) [inline Changelog]: cut of/cut for/

File: Changelog [hunk 7]
```diff
@@ -1,6 +1,87 @@
...
+  "Open base" when viewing the list of bases at a Location.
+  (nthykier)
+- Base purchase screen now provides more details about the base.
+  Including accurate purchase and maintenance cost plus detection
```
Comment (sunfall) [inline Changelog]: (here and line above) s/base. Including/base, including/

File: Changelog [hunk 8]
```diff
@@ -1,6 +1,87 @@
 Endgame: Singularity Changelog
 
+GAME CONTENT CHANGES:
```
Comment (sunfall) [inline Changelog]: A broad note: it looks like the old changelog entries (mostly) consistently didn't end with a period.  That's a bit bleh, but we should be consistent.  Either remove periods at the end of entries here, or add them elsewhere.

File: Changelog [hunk 9]
```diff
@@ -1,6 +1,87 @@
...
+- Theming support to change background image, colors, fonts, etc.
+  The themes can have language specific content (e.g. fonts better
+  suited for the given language) (MestreLion, Xenega)
+- Support changing language without restarting the game.
+  (Xenega)
+- Add warnings for some common issues such as when the player only
+  has one base left or that when changing CPU of a base will remove
+  existing CPU (Closes: #17) (Xenega)
+- Support renaming bases (Xenega)
+- Add a 24hour finance report (Xenega)
```
Comment (sunfall) [inline Changelog]: s/24hour/24-hour/

File: Changelog [hunk 10]
```diff
@@ -13,9 +94,23 @@ USER-VISIBLE CHANGES:
 - Updated es_AR translation (Closes: #20)(justapawn)
 - Try to use user's locale, or a similar language, as game language (MestreLion)
 - Launcher for Linux is now "singularity"
-- Revamped Options screen, including new resolutions (MestreLion)
+- Revamped Options screen, including new resolutions (MestreLion, Xenega)
 
 OTHER CHANGES:
+- New savegame format based on json to avoid a savegame format that can include
```
Comment (sunfall) [inline Changelog]: s/json/JSON/

File: Changelog [hunk 11]
```diff
@@ -13,9 +94,23 @@ USER-VISIBLE CHANGES:
...
+  arbitrary code execution via Python's pickle module.  Please note that any old
+  style savegame will still trip the picke-based codepaths already in the
+  savegame listing to simplify the process for players upgrading from 0.30c.
+  (Closes: #52)  (nthykier)
+- Add basic documentation of the data files in the docs folder of the github
```
Comment (sunfall) [inline Changelog]: s:docs:docs/:
s/github/Github/

File: Changelog [hunk 12]
```diff
@@ -13,9 +94,23 @@ USER-VISIBLE CHANGES:
...
+  repo (nthykier)
+- Work around a file descriptor "leak" in pygame.  This fix requires pygame
```
Comment (sunfall) [inline Changelog]: s/pygame/Pygame/ (twice)

File: Changelog (hunk 4 above)
Comment (nthykier) [reply Changelog]: Correct - `under consideration` is for purchases you have *not* (yet) committed  too.

File: Changelog [hunk 13]
```diff
@@ -1,21 +1,123 @@
...
+- Show standard costs and detection chances of items and bases
+  in the Knowledge screen excluding any location modifiers
+  (Xenega)
+- Show location modifiers more prominently in the location screen
+  and the dialog for constructing a new base (Xenega)
+- Show difficulty setting and in-game time for new savegames
+  (Xenega)
+- Loading an old savegame (e.g. from 0.30c) will now
+  retroactively apply the game data from the latest version
+  of singularity - including making new game content
```
Comment (sunfall) [inline Changelog]: s/singularity/Singularity

File: Changelog [hunk 14]
```diff
@@ -1,21 +1,123 @@
...
+  researchable (Closes: #56, #122, #145) (nthykier)
+- Unify multiple messages into a single dialog with multiple
+  pages.  This new dialog also supports resuming the game
+  instead of forcing the game to be paused (Xenega)
+- Events can now expire and two non-unique events now do
```
Comment (sunfall) [inline Changelog]: Missing period at end of this line.

File: Changelog [hunk 15]
```diff
@@ -1,21 +1,123 @@
...
+  characters (Closes: https://bugs.debian.org/718447)
+  (nthykier)
+- Fix visual lag at the highest game speed due to recalculating
+  the "day/night" layers (nthykier)
+- It is now possible to use double-click as a short-cut for
+  "Open base" when viewing the list of bases at a Location.
+  (nthykier)
+- Base purchase screen now provides more details about the base,
+  including accurate purchase and maintenance cost plus detection
+  rates (at the level the player is allowed to see them).
```
Comment (sunfall) [inline Changelog]: Extraneous period at the end here (yeah, this is nitpicky, feel free to ignore :P)

File: Changelog [hunk 16]
```diff
@@ -1,21 +1,123 @@
...
+  (Closes: #53) (Xenega)
+- Support deleting existing savegames via the Load screen
+  (Closes: #48) (Xenega)
+- Select the amount of CPU at the same dialog as the CPU selection
+  dialog (Xenega)
+- Add new "nightmode" theme  (Closes: #119) (cryptarch)
+- Theming support to change background image, colors, fonts, etc.
+  The themes can have language specific content (e.g. fonts better
+  suited for the given language) (MestreLion, Xenega)
+- Support changing language without restarting the game.
```
Comment (sunfall) [inline Changelog]: Extraneous period (see above).

File: Changelog [hunk 17]
```diff
@@ -1,21 +1,123 @@
 Endgame: Singularity Changelog
 
+GAME CONTENT CHANGES:
+- Endgame requires a lot less grinding (emh, Xenega)
+- New early-midgame bases to make the progression into the
+  midgame more smooth (Xenega)
+- All CPUs have been rebalanced.  Notably midgame CPUs have
+  a much better cost/effeciency ratio and endgame CPUs have
+  been nerfed considerably (Xenega)
+- New items security and reactor items are now available
```
Comment (sunfall) [inline Changelog]: "New security and reactor items are now available"

**Pull request merged**